source/includes/connection-snippets/scram-snippets/c-connection.c

```diff
@@ -0,0 +1,25 @@
+// begin scram connection
```

Comment:
Is the purpose of all of these examples to _not_ specify the `authMechanism`? IIRC, modern drivers can infer whether to use SCRAM-SHA-1 or SCRAM-SHA-256 for a particular user, since 256 was introduced in MongoDB 4.0, so perhaps this was intentional.

Reply:
This snippet was actually taken verbatim from https://docs.mongodb.com/ecosystem/drivers/c/#connect-to-mongodb-atlas (all the SCRAM connection snippets were taken verbatim from the current docs). I can't speak to why the original developer who made this snippet didn't explicitly specify `authMechanism` but the default is SCRAM-SHA-1 and you can specify to use SCRAM-SHA-256 if needed. 

However, I don't see the need to specify this explicitly for these snippets, and I would prefer these snippets to be as consistent to the adjacent driver docs as possible.